core/string/ustring.cpp

```diff
@@ -3371,6 +3371,145 @@ float String::similarity(const String &p_string) const {
 	return (2.0f * inter) / sum;
 }
 
+// Based on: https://github.com/ka-weihe/fastest-levenshtein
+int String::levenshtein_distance(const String &p_string) const {
+	auto a = *this;
+	auto b = p_string;
```

Comment:
We do not use `auto` in the Godot codebase (with a few exceptions), so this should be `const String &`.